.github/workflows/ci-pr.yaml

```diff
@@ -15,8 +15,10 @@
 name: "Continuous Integration - Pull Request"
 on:
   pull_request:
+    types: [review_requested, ready_for_review]
```

Comment:
Question: Could you share the link to the relevant GitHub documentation?

Reply:
sure thing https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request